customer_data_platform/authorized_api/audiences.rst

```diff
@@ -0,0 +1,12 @@
+Audiences API
+=============
+
+.. raw:: html
+
+    <div id='redoc-container'>
+    </div>
+    <script>
+        (function() {
+            Redoc.init('../../../_static/api/customer_data_platform_audiences_api.json', {}, document.getElementById('redoc-container'), () => {window.prepareRedocMenu ? window.prepareRedocMenu() : setTimeout(()=>{window.prepareRedocMenu()}, 2000)});
```

Comment:
one ../ too many? checked on built version and its not working

Reply:
https://developers.piwik.pro/en/ppcdp-860/customer_data_platform/authorized_api/audiences.html - check it yourself

Reply:
now points to https://developers.piwik.pro/en/_static/api/customer_data_platform_audiences_api.json
instead of https://developers.piwik.pro/en/ppcdp-860/_static/api/customer_data_platform_audiences_api.json

Reply:
I tested it locally and it worked. :shrug: Fixed